<?xml version="1.0" encoding="UTF-8"?>
<testng-results total="7" passed="6" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2021-06-03T22:25:43Z" name="Default suite" finished-at="2021-06-03T22:25:54Z" duration-ms="10796">
    <groups>
    </groups>
    <test started-at="2021-06-03T22:25:43Z" name="Default test" finished-at="2021-06-03T22:25:54Z" duration-ms="10796">
      <class name="Capstone.Login_Home_Page">
        <test-method signature="Login()" started-at="2021-06-03T22:25:43Z" name="Login" finished-at="2021-06-03T22:25:53Z" duration-ms="10241" status="PASS">
        </test-method>
        <test-method signature="HomePageVerifyTitle()" started-at="2021-06-03T22:25:53Z" name="HomePageVerifyTitle" finished-at="2021-06-03T22:25:53Z" duration-ms="25" status="PASS">
        </test-method>
        <test-method signature="HomePageVerifyLogo()" started-at="2021-06-03T22:25:53Z" name="HomePageVerifyLogo" finished-at="2021-06-03T22:25:53Z" duration-ms="101" status="PASS">
        </test-method>
        <test-method signature="HomePageDropDownsCount()" started-at="2021-06-03T22:25:53Z" name="HomePageDropDownsCount" finished-at="2021-06-03T22:25:53Z" duration-ms="36" status="PASS">
        </test-method>
        <test-method signature="HomePageDropDownsName()" started-at="2021-06-03T22:25:53Z" name="HomePageDropDownsName" finished-at="2021-06-03T22:25:54Z" duration-ms="202" status="PASS">
        </test-method>
        <test-method signature="VerifyVisitTagHomePagePresence()" started-at="2021-06-03T22:25:54Z" name="VerifyVisitTagHomePagePresence" finished-at="2021-06-03T22:25:54Z" duration-ms="77" status="PASS">
        </test-method>
        <test-method signature="VerifyVisitTagHomePageName()" started-at="2021-06-03T22:25:54Z" name="VerifyVisitTagHomePageName" finished-at="2021-06-03T22:25:54Z" duration-ms="79" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected:<null> but was:<Visit>]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected:<null> but was:<Visit>
	at org.testng.Assert.fail(Assert.java:89)
	at org.testng.Assert.failNotEquals(Assert.java:480)
	at org.testng.Assert.assertEquals(Assert.java:118)
	at org.testng.Assert.assertEquals(Assert.java:171)
	at org.testng.Assert.assertEquals(Assert.java:181)
	at Capstone.Login_Home_Page.VerifyVisitTagHomePageName(Login_Home_Page.java:106)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:564)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:74)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:673)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:846)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1170)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.runWorkers(TestRunner.java:1147)
	at org.testng.TestRunner.privateRun(TestRunner.java:749)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:317)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:312)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:274)
	at org.testng.SuiteRunner.run(SuiteRunner.java:223)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1039)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:964)
	at org.testng.TestNG.run(TestNG.java:900)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception>
        </test-method>
        <test-method is-config="true" signature="after_suite()" started-at="2021-06-03T22:25:54Z" name="after_suite" finished-at="2021-06-03T22:25:55Z" duration-ms="992" status="PASS">
        </test-method>
      </class>
    </test>
  </suite>
</testng-results>
